### ISSUE TITLE
The terms "alternate" and "alternative" are used inconsistently through the spec and non-normative material

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#4135]
During a review of content modified as part of a PR to update the WCAG 2.1 content with appropriate updates from 2.2, [I identified some pre-existing inconsistent uses](https://github.com/w3c/wcag/pull/4007#issuecomment-2454854511) of the terms "alternate" and "alternative.

In the normative text, use of the term “alternate” is in the minority of occurrences, but it does get used in the defined term "conforming alternate versions". There are many occurrences of the word "alternative" in normative text and definitions, including:

- [Text Alternatives](https://www.w3.org/TR/WCAG22/#text-alternatives)
- [Media Alternative](https://www.w3.org/TR/WCAG22/#media-alternative-prerecorded)
- [media alternative for text](https://www.w3.org/TR/WCAG22/#dfn-media-alternative-for-text)
- [alternative for time-based media](https://www.w3.org/TR/WCAG22/#dfn-alternative-for-time-based-media)
-  Alternative method ( in [Accessible Authentication](https://www.w3.org/TR/WCAG22/#accessible-authentication-minimum))
-  [alternative presentation](https://www.w3.org/TR/WCAG22/#cc2)
- [alternative input methods](https://www.w3.org/TR/WCAG22/#dfn-assistive-technologies)
- [alternative keyboards](https://www.w3.org/TR/WCAG22/#dfn-assistive-technologies)
- [alternative pointing devices](https://www.w3.org/TR/WCAG22/#dfn-assistive-technologies)

Even within the definition of “alternate conforming version”, [note 5](Note 5
The conforming alternative version does not need to reside within the scope of conformance, or even on the same Web site, as long as it is as freely available as the non-conforming version.) refers to “the conforming alternative version”. 

WCAG treats these words as synonyms, although "alternative" is the more accurate term, which cannot be misconstrued. The question is, do we want to try to settle on one term and use it consistently?

**Comment by mbgower** [comment]
BTW, the intermixing of the terms is even more pronounced in the non-normative documents such as [Understanding Conforming Alternate Versions](https://www.w3.org/WAI/WCAG21/Understanding/conformance#conforming-alt-versions):

> Conformance requirement #1 allows non-conforming pages to be included within the scope of conformance as long as they have a "conforming alternate version". The conforming alternative version is defined as...

> For the purpose of determining conformance, alternatives to part of a page's content are considered part of the page when the alternatives can be obtained directly from the page, e.g., a long description or an alternative presentation of a video.

**Comment by GreggVan** [comment]
I  think it is inadvertent, and unless someone knows otherwise - I would think a global search and replace for the phrase is in order.

**Comment by patrickhlauke** [comment]
I'd be happy to see this normalised to "alternative" throughout (even though I'm guessing this will cause a bit of pain as it touches on normative text, despite being an editorial-only change)

**Comment by bruce-usab** [comment]
FWIW, "alternate formats" was a term used early on with U.S. disability civil rights regs.  Before even "electronic and information technology" (EIT).  Maybe even before "alt text" was coined.  I speculate that is why it was Conforming Alternate Version in 2.0.

I agree that "alternative" is better and that a global search and replace for the phrase is in order for the non-normative docs.

**Comment by detlevhfischer** [comment]
@bruce-usab since "alternate" is in the wording of 5.2.1 Conformance level and in the glossary, would it not male more sence to change occurences of "alternative" to "alternate" (just to keep normative and informative wording in line)?

**Comment by patrickhlauke** [comment]
I think the tricky part will be that terms like "text alternative" are the more common/used variant (even in other specs, like the HTML spec). also, i don't think you could/should outright replace "text alternative" with "text alternate", as "alternate" as a noun (rather than as an adjective)  is - at least to my knowledge - not that common and feels very archaic.

**Comment by bruce-usab** [comment]
There is one (and only one) _normative_ instance of alternative that should be changed to alternate.

My strong preference is for that one to be corrected and that the informative documents use "alternative" when either word would fit.  I do not think we should make a push to have "conforming alternate version" become "conforming alternative version".

**Comment by mbgower** [comment]
Closed after discussion in TF.
